modules/ROOT/pages/compatibility-issues-runtime-java.adoc

```diff
@@ -0,0 +1,15 @@
+= Troubleshooting Issues with Mule Runtime and Java
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The modules in your project can have compatibility issues with Java or Mule runtime.
```

Comment:
Should "have" be "experience"? Or maybe the modules can be incompatible?

Reply:
The module can be incompatible with the Java version, with the Mule runtime version, or with both.
Why would you choose experience? 
It can be the case that a module is no longer compatible after changing the Java version in a project, for example.
Did I answer your question?

Reply:
OK, I guess I was just looking for a more precise word than "have".

Reply:
OK! Thanks!